### PR TITLE
Add Poochy & Yoshi's Woolly World

### PR DESCRIPTION
**Pull request by Krusher97**
Adds Poochy and Yoshi's Woolly World.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by Krusher97** [comment]
I borked up the name. The "and" should be "&"

File: games/poochy-and-yoshis-woolly-world/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Poochy & Yoshi's Woolly World"
+description = "Poochy & Yoshi's Woolly World is a platforming game for Nintendo 3DS. It is the second port of the series of platform games starring Yoshi, being a port of Yoshi's Woolly World, which was released for the Wii U in 2015. The game includes the entirety of the original game's levels, along with some bonus features."
+needs_system_files = false
```
Comment (Leo626) [inline games/poochy-and-yoshis-woolly-world/game.dat]: There's an [issue](https://github.com/citra-emu/citra/issues/3321) for this game that should be added between `description` and `needs_system_files`. See [contributing](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md) guide for more info on this.

**Comment by Krusher97** [comment]
seems like the browser ate one screenshot while submitting the entry. How do I add another screenshot now?

**Comment by Leo626** [comment]
The only thing remains is that the wiki is a bit lacking. Should mention audio accuracy and stability whether it crashes or not, similar to the example in the contributing guide.

File: games/poochy-and-yoshis-woolly-world/game.dat [hunk 2]
```diff
@@ -0,0 +1,10 @@
+title = "Poochy & Yoshi's Woolly World"
+description = "Poochy & Yoshi's Woolly World is a platforming game for Nintendo 3DS. It is the second port of the series of platform games starring Yoshi, being a port of Yoshi's Woolly World, which was released for the Wii U in 2015. The game includes the entirety of the original game's levels, along with some bonus features."
+github_issues = [3055] [3321]
```
Comment (Leo626) [inline games/poochy-and-yoshis-woolly-world/game.dat]: This not the correct way. See this [example.](https://github.com/citra-emu/citra-games-wiki/blob/master/games/pokemon-ultra-sun-ultra-moon/game.dat#L3)

**Pull request merged**